From 87bd9ff4a973c1b358af1f0c70efc24b2da78c39 Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 08:49:36 -0700
Subject: [PATCH 08/96] HBASE-6689 HRegion.increment comment nits

Reason: Supportability
Author: Gregory Chanan
Ref: CDH-5549
(cherry picked from commit 442578ccc70521c7a684eb419cc9b50af5105ffb)
---
 .../apache/hadoop/hbase/regionserver/HRegion.java  |    2 +-
 .../hadoop/hbase/regionserver/TestHRegion.java     |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java b/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
index 593b4ba..62cedf0 100644
--- a/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
+++ b/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
@@ -5176,7 +5176,7 @@ public class HRegion implements HeapSize { // , Writable{
                 now, Bytes.toBytes(amount));
             kvs.add(newKV);
 
-            // Append update to WAL
+            // Prepare WAL updates
             if (writeToWAL) {
               if (walEdits == null) {
                 walEdits = new WALEdit();
diff --git a/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java b/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java
index dec7fc5..e56e49e 100644
--- a/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java
+++ b/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java
@@ -4083,7 +4083,7 @@ public class TestHRegion extends HBaseTestCase {
     this.region = initHRegion(tableName, method, conf, family);
     final HRegion region = this.region;
     final AtomicBoolean incrementDone = new AtomicBoolean(false);
-    Runnable reader = new Runnable() {
+    Runnable flusher = new Runnable() {
       @Override
       public void run() {
         while (!incrementDone.get()) {
@@ -4101,7 +4101,7 @@ public class TestHRegion extends HBaseTestCase {
     int incCounter = 100;
     long expected = threadNum * incCounter;
     Thread[] incrementers = new Thread[threadNum];
-    Thread flushThread = new Thread(reader);
+    Thread flushThread = new Thread(flusher);
     for (int i = 0; i < threadNum; i++) {
       incrementers[i] = new Thread(new Incrementer(this.region, incCounter));
       incrementers[i].start();
-- 
1.7.0.4

